File: elearning-service/src/main/java/com/app/elearningservice/dto/EnrollmentDTO.java
package com.app.elearningservice.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class EnrollmentDTO {
    private Long timeTableId;
    private Long userId;
    private Long courseId;
    private String status;
    private String email;
    private String time;
    private String courseName;
    private String thumbnail;
    private String createdDate;
    private String updatedDate;
}

File: elearning-ui/src/app/shared/model/EnrollmentDTO.ts
export class EnrollmentDTO {
  constructor(
    public timeTableId: number = -1,
    public courseId: number = -1,
    public userId: number = -1,
    public time: string = '',
    public status: string = '',
    public email: string = '',
    public courseName: string = '',
    public thumbnail: string = '',
    public createdDate: string = '',
    public updatedDate: string = '',
  ) {
  }
}
